packages/core/package.json

```diff
@@ -86,6 +85,7 @@
   },
   "devDependencies": {
     "@theia/ext-scripts": "^0.5.0",
+    "@types/electron-store": "^1.3.1",
```

Comment:
Should not it go to `dependencies` of `@theia/electron`? then it will be available everywhere in this repo and to extenders.

Reply:
@marechal-p I think this is a new dependency; a CQ will be needed for a dev-dependency (covering this version and later), or I'll do the new process if we add it as a production dependency.

Reply:
@marcdumais-work CQ is already approved apparently for this package as a devdep?
https://github.com/theia-ide/theia/pull/4869#discussion_r278124510

@akosyakov Indeed, at first I was afraid I would lose typings since `@theia/core` does not depend on it, but since it is in the same yarn workspace it seems to be fine, thanks!